GameData/ROCapsules/PartConfigs/Apollo D2/D2CommandModule.cfg

```diff
@@ -0,0 +1,452 @@
+	//	Animations and Textures
+	//	============================================================================
+
+	EFFECTS
+	{
+		rcs
+		{
+			AUDIO_MULTI_POOL
+			{
+				channel = Ship
+				transformName = rcsTransform
+				clip = sound_rocket_mini
+				volume = 0.0 0.0
+				volume = 0.02 0.1
+				volume = 0.5 0.1
+				volume = 1.0 0.1
+				pitch = 0.0 0.75
+				pitch = 1.0 1.5
+				loop = true
+			}
+			MODEL_MULTI_PARTICLE
+			{
+				modelName = Squad/FX/Monoprop_small
+				transformName = rcsTransform
+				emission = 0.0 0.0
+				emission = 0.1 0.0
+				emission = 1.0 1.0
+				speed = 0.0 0.8
+				speed = 1.0 1.0
+				localRotation = -90, 0, 0
+			}
+		}
+	}
+
+	//	============================================================================
+	//	Comms
+	//	============================================================================
+
+	MODULE:NEEDS[!RemoteTech]
+	{
+		name = ModuleDataTransmitter
+		antennaType = DIRECT
+		antennaPower = 64
+		antennaCombinable = True
+		antennaCombinableExponent = 2.0
+		packetInterval = 1.0
+		packetSize = 2.0
+		requiredResource = ElectricCharge
+		packetResourceCost = 0.04
+		DeployFxModules = 0
+	}
+
+	MODULE:NEEDS[RemoteTech]
+	{
+		name = ModuleSPU
+		IsRTCommandStation = False
+		RTCommandMinCrew = 0
+	}
+
+	MODULE:NEEDS[RemoteTech]
+	{
+		name = ModuleRTAntenna
+		ActionMode0Name = Deactivate Internal Antenna
+		ActionMode1Name = Activate Internal Antenna
+		IsRTActive = False
+		Mode0OmniRange = 0
+		Mode1OmniRange = 400000
+		EnergyCost = 0.02
+		DeployFxModules = 0
+
+		TRANSMITTER
+		{
+			PacketInterval = 0.4
+			PacketSize = 0.27
+			PacketResourceCost = 0.025
+		}
+	}
+}
+
+//	================================================================================
+//	Configure RealAntennas
+//	================================================================================
+
+@PART[ROC-D2CM]:BEFORE[RealAntennas]
+{
+	!MODULE[ModuleDataTransmitter],* {}
+	MODULE
+	{
+		name = ModuleRealAntenna
+		referenceGain = 3.0		//Guess based on Block II
+		RFBand = S		//Default to S-band
+	}
+}
+
+//	================================================================================
+//	Stop Capsule Going Kaboom During Reentry
+//	================================================================================
+
+@PART[ROC-D2CM]:AFTER[zzzRealismOverhaul]
+{
+	%maxTemp = 900
+	%skinMaxTemp = 3600
+}
+
+// Specific adjustments after default setup
+@PART[ROC-D2CM]:AFTER[RealismOverhaul_HeatShield]
+{
+	@MODULE[ModuleAblator]
+	{
+		charModuleName = HeatShieldCharAnim
+		charMin = 0
+		charMax = 1
+	}
+}
+
+//	==================================================
+//	D-2 command module.
+
+//	TAC Life Support compatibility.
+//	==================================================
+
+@PART[ROC-D2CM]:AFTER[RealismOverhaul]:NEEDS[TacLifeSupport]
+{
+	@description ^= :$: Supports a crew of 3 for 6 days.:
+
+	@MODULE[ModuleFuelTanks]
+	{
+		@volume += 300
```

Comment:
Don't give extra volume just because TACLS is installed.  Give the part the full volume and then consume some if TACLS needs to.

Reply:
will fix